en/identity-server/next/docs/guides/multitenancy/manage-tenants.md

```diff
@@ -1,27 +1,41 @@
-# Tenant management
+# Manage Root Organizations (Tenants)
 
-By default,  {{ product_name }} is configured with a single tenant named `carbon.super`. If you have the required permissions, you can configure more tenant domains and maintain multiple isolated user bases on a single server/cluster at the same time.
+!!! Note
+    From now on, **Tenants** will be referred to as **Root Organizations**.
+
+    This change is to ensure consistency across our documentation and to better reflect the hierarchical structure of our organization system.
+
+By default,  {{ product_name }} is configured with a single root organization named `carbon.super`. If you have the required permissions, you can configure more root organizations and maintain multiple isolated user bases on a single server/cluster at the same time.
```

Comment:
```suggestion
By default, {{ product_name }} is configured with a single root organization named `carbon.super`. If you have the required permissions, you can configure more root organizations and maintain multiple isolated user bases on a single server/cluster at the same time.
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4820/commits/129149a8ed3f189a6d69a63b37780762503ebf0a